### PR TITLE
Add links and helper command for Web Action URL.

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,11 +107,22 @@ If you are using the provided workspace.json, use Weather Company Data to provid
 Follow this link and hit `Create`:
 * [**Weather Company Data**](https://console.ng.bluemix.net/catalog/services/weather-company-data)
 
-This service includes an OpenWhisk package. Run the following command to update your OpenWhisk bindings:
+This service includes an OpenWhisk package.
+Run the following to [install the OpenWhisk bindings for IBM cloud](https://console.bluemix.net/openwhisk/learn/cli):
```
Comment (markstur) [inline README.md]: s/cloud/Cloud/

File: README.md [hunk 2]
```diff
@@ -107,11 +107,22 @@ If you are using the provided workspace.json, use Weather Company Data to provid
...
+
+Run the following command to update your OpenWhisk bindings if they are already installed:
 ```
 $ bx wsk package refresh
 ```
 
+Run the following to test OpenWhisk on IBM Cloud:
+```
+$ bx wsk action invoke /whisk.system/utils/echo -p message hello --result
+Copy
```
Comment (markstur) [inline README.md]: Is the line "Copy" here just a copy/paste error?

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md]: yes

File: README.md (hunk 1 above)
Comment (markstur) [reply README.md]: don't see it

File: README.md (hunk 2 above)
Comment (markstur) [reply README.md]: merge then fix

**Pull request merged**